### PR TITLE
Update coroutine-context-and-dispatchers.md

### PR DESCRIPTION
**Pull request by BattleShipPark**
I found an uncompleted sentence.

### CODE REVIEW

File: docs/topics/coroutine-context-and-dispatchers.md [hunk 1]
```diff
@@ -65,8 +65,7 @@ context of the main `runBlocking` coroutine which runs in the `main` thread.
 [Dispatchers.Unconfined] is a special dispatcher that also appears to run in the `main` thread, but it is,
 in fact, a different mechanism that is explained later.
 
-The default dispatcher that is used when no other dispatcher is explicitly specified in the scope.
```
Comment (dkhalanskyjb) [inline docs/topics/coroutine-context-and-dispatchers.md:68]: I think the correct thing to do here is to remove the word `that`. If we remove the sentence, the (as is, incorrectly stated) piece of information that there exists a default fallback dispatcher will be lost.

Comment (BattleShipPark) [reply docs/topics/coroutine-context-and-dispatchers.md:68]: We have two terms: a default dispatcher and Dispatchers.Default.

For default dispatcher, its explanation is done before this line like this:
> When launch { ... } is used without parameters, it inherits the context (and thus dispatcher) from the CoroutineScope it is being launched from. In this case, it inherits the context of the main runBlocking coroutine which runs in the main thread.

In the line, the document need to explain Dispatchers.Default.

Comment (dkhalanskyjb) [reply docs/topics/coroutine-context-and-dispatchers.md:68]: > For default dispatcher, its explanation is done before this line like this

This is not the case. The explanation that you cited refers to inherited contexts. The default dispatcher is `Dispatchers.Default`. Please see the docs for `Dispatchers.Default` or, if you're still not convinced, this code that is referenced throughout the codebase: https://github.com/Kotlin/kotlinx.coroutines/blob/master/kotlinx-coroutines-core/jvm/src/CoroutineContext.kt#L32-L37

Comment (BattleShipPark) [reply docs/topics/coroutine-context-and-dispatchers.md:68]: Good. I misunderstood the default dispatcher. Thanks 👍 
Do I need to send PR again removing only the word "that"?

Comment (qwwdfsad) [reply docs/topics/coroutine-context-and-dispatchers.md:68]: Adjusting this PR should be enough

**Pull request closed without merging**